backend/src/main/java/com/celuveat/restaurant/query/dao/support/RestaurantLikeQueryDaoSupport.java

```diff
@@ -15,4 +17,10 @@ public interface RestaurantLikeQueryDaoSupport extends JpaRepository<RestaurantL
     List<RestaurantLike> findAllByMemberIdOrderByCreatedDateDesc(Long memberId);
 
     Integer countByRestaurant(Restaurant restaurant);
+
+    @Query("SELECT rl FROM RestaurantLike rl WHERE rl.member.id = :memberId AND rl.restaurant.id IN :restaurantIds")
+    List<RestaurantLike> findAllByMemberIdAndRestaurantIdsIn(
+            @Param(("memberId")) Long memberId,
+            @Param("restaurantIds") List<Long> restaurantIds
+    );
```

Comment:
```suggestion
    List<RestaurantLike> findAllByMemberIdAndRestaurantIdIn(
            Long memberId,
            List<Long> restaurantIds
    );
```
이렇게도 되는 것 같은데 `support`를 최대한 활용하는 방향으로 쓰는게 어때?